src/renderer/public/localization/locales/es/translation.json

```diff
@@ -1,15 +1,18 @@
 {
+  "alwaysOnTop": "Always on Top?",
```

Comment:
To be translated?

Reply:
Yep I couldn’t find the right wording that made sense

Reply:
```suggestion
  "alwaysOnTop": "Siempre en la cima?",
```